runtime/shared_common/shrinit.cpp

```diff
@@ -3516,6 +3516,7 @@ j9shr_init(J9JavaVM *vm, UDATA loadFlags, UDATA* nonfatal)
 		}
 
 		config->getCacheSizeBytes = j9shr_getCacheSizeBytes;
+		config->getSharedClassCacheMode = j9shr_getSharedClassCacheMode;
 		config->getTotalUsableCacheBytes = j9shr_getTotalUsableCacheBytes;
```

Comment:
It's easier to see that all fields are initialized if they are set in the same order as they are declared. Please swap these two lines.